transform.js

```diff
@@ -1 +1 @@
-module.exports = require("./lib/_stream_transform.js")
+module.exports = require('./readable').Transform
```

Comment:
could this theoretically pull in more then it did previously? probably just passthrough right